libs/portals/admin/signature-collection/src/screens/AllLists/index.tsx

```diff
@@ -34,37 +34,54 @@ const Lists = () => {
   const [page, setPage] = useState(1)
   const [filters, setFilters] = useState<Filters>({
     area: [],
+    candidate: [],
     input: '',
   })
+  const [candidates, setCandidates] = useState<
+    Array<{
+      label: string
+      value: string
+    }>
+  >([])
 
   useEffect(() => {
-    if (filters.area.length > 0 || filters.input.length > 0) {
-      let filteredList = []
+    let filteredList: SignatureCollectionList[] = allLists
 
-      //filter by area
-      filteredList = allLists.filter((list) => {
-        return filters.area.some((filter) => filter === list.area.name)
-      })
-
-      //filter by input
-      filteredList = (filteredList.length > 0 ? filteredList : allLists).filter(
-        (list) => {
-          return (
-            list.owner.name
-              .toLowerCase()
-              .includes(filters.input.toLowerCase()) ||
-            list.area.name.toLowerCase().includes(filters.input.toLowerCase())
-          )
-        },
+    filteredList = filteredList.filter((list) => {
+      return (
+        // Filter by area
+        (filters.area.length === 0 || filters.area.includes(list.area.name)) &&
```

Comment:
Might want to consider creating constant for these like const FilterArea = filters.area etc...